arcgis_map_sdk_ios/ios/Classes/ArcgisMapView.swift

```diff
@@ -190,6 +191,24 @@ class ArcgisMapView: NSObject, FlutterPlatformView {
             result(success)
         }
     }
+    
+    private func onMoveCameraToPoints(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
+        let dict = call.arguments as! Dictionary<String, Any>
+    
+        let payload: MoveToPointsPayload = try! JsonUtil.objectOfJson(dict)
+        let polyline = AGSPolyline(points: payload.points.map { latLng in AGSPoint(x: latLng.longitude,y:latLng.latitude,spatialReference:.wgs84())
```

Comment:
I quickly formatted this for you
```suggestion
        let polyline = AGSPolyline(points: payload.points.map { latLng in AGSPoint(x: latLng.longitude, y:latLng.latitude, spatialReference: .wgs84())
```